src/i965_drv_video.c

```diff
@@ -6720,6 +6720,7 @@ i965_Terminate(VADriverContextP ctx)
     int i;
 
     if (i965) {
+#if HAVE_HYBRID_CODEC
```

Comment:
Context and reasoning to support your change will make other developers think twice before attempting to revert it in the future ;-)